cinema/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from rest_framework.urls import app_name
```

Comment:
The import `from rest_framework.urls import app_name` is unnecessary because `app_name` is a standard Django feature and does not need to be imported from `rest_framework.urls`. You can remove this line.